src/Components/UI/videoformmodal/VideoFormModal.jsx

```diff
@@ -1,117 +1,205 @@
-        placeholder="7.8"
-        onChange={(value) => handleVideoChange('rating', value)}
-      />
-      <br />
-      <Input
-        id="runtime"
-        value={video.runtime}
-        lable="runtime"
-        placeholder="minutes"
-        onChange={(value) => handleVideoChange('runtime', value)}
-      />
-      <br />
-      <label htmlFor="overview">overview</label>
-      <textarea id="overview" value={video.description} onChange={(e) => setVideo({ ...video, description: e.target.value })} placeholder="Movie description" />
-      <br />
-      <MyButton onClick={reset} className="button__gray-blurred">
-        reset
-      </MyButton>
-      <MyButton onClick={addNewVideo} className="button__red">
-        submit
-      </MyButton>
-    </form>
-
+    <>
+      {request.error ? <p>Error on request</p> : ''}
+      <Formik
+        initialValues={video}
+        validate={validation}
+        onSubmit={onSubmitFormic}
+        enableReinitialize
+      >
+        {({ isSubmitting, resetForm }) => (
+          <Form>
+            <ErrorMessage name="title" component="div" />
+            <label style={{ color: 'red' }} htmlFor="title">Title</label>
+            <Field
+              id="title"
+              type="text"
+              name="title"
+              placeholder="Enter movie title"
+            />
+            <br />
+            <ErrorMessage name="releaseDate" component="div" />
+            <label style={{ color: 'red' }} htmlFor="releaseDate">RELEASE DATE</label>
+            <Field
+              id="releaseDate"
+              type="text"
+              name="releaseDate"
+              placeholder="Select Date"
+            />
+            <br />
+            <ErrorMessage name="posterPath" component="div" />
+            <label style={{ color: 'red' }} htmlFor="posterPath">MOVIE URL</label>
+            <Field
+              id="posterPath"
+              type="text"
+              name="posterPath"
+              placeholder="https://"
+            />
+            <br />
+            <ErrorMessage name="voteAverage" component="div" />
+            <label style={{ color: 'red' }} htmlFor="voteAverage">RATING</label>
+            <Field
+              id="voteAverage"
+              type="number"
+              name="voteAverage"
+              placeholder="7.8"
+            />
+            <br />
+            <ErrorMessage name="genres" component="div" />
+            <label style={{ color: 'red' }} htmlFor="voteAverage">GENRE</label>
+            <Field
+              id="genres"
+              type="text"
+              name="genres"
+              placeholder="Select Genre"
+            />
+            <br />
+            <ErrorMessage name="runtime" component="div" />
+            <label style={{ color: 'red' }} htmlFor="runtime">RUNTIME</label>
+            <Field
+              id="runtime"
+              type="number"
+              name="runtime"
+              placeholder="minutes"
+            />
```

Comment:
NiT: lots of html dulpication... maybe we could have just const with array of fields information and iterate trhouh it here